### PR TITLE
Created agenda card and agenda carousel

### PR DESCRIPTION
**Pull request by IvarSchuyt**
## What does this change?

Resolves issue #58 and #55

Agenda card:
I have issues importing data into a component, thats why i already set it up for use and added some dummy data for now.

Agenda carrousel: 
Responsive carrousel with scroll snap. I have copied the card into this component a few times, because of the data issue. This will of course be fixed when i know how to import data correctly.
I am trying to PE this carrousel with scroll indicators, but i'm having some trouble with this as well. This is why I commented out the code that is not yet working.

[livesite](https://buurtcampus-oost-ivar.vercel.app/)

## How Has This Been Tested?

- [x] User test
- [x] Accessibility test
- [ ] Performance test
- [x] Responsive Design test

## Images
Screenshot 2024-05-17 at 13 53 36

## How to review
Please try and test the accessibility of this component. I have already seen that width of the carrousel needs to be adjusted on mobile view. I'm also looking forward to critique on my code.

### CODE REVIEW

File: src/lib/molecules/agenda-card.svelte [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<script>
+	import { Button } from '$lib/index.js';
+
+	export let cardEvent, cardDate, cardAddress, cardPrice, cardImage;
+</script>
+
+<section style="background-image: {cardImage} ;">
+	<!-- <h2>{cardEvent}</h2> -->
+	<div class="data">
```
Comment (ju5tu5) [inline src/lib/molecules/agenda-card.svelte:9]: I get this div.. you want to group these elements together..

File: src/lib/molecules/agenda-carousel.svelte [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+		{
+			event: '{event}',
+			date: '{date}',
+			address: '{address}',
+			price: '{price}',
+			image: '{image}'
+		}
+	];
+
+	// onMount(async () => {
```
Comment (ju5tu5) [inline src/lib/molecules/agenda-carousel.svelte:16]: Remove the comment if you're not using it.. production shouldn't have commented code..

File: src/lib/molecules/agenda-carousel.svelte [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+	// });
+	// console.log(carrousel, indicator);
+	// });
+</script>
+
+<div class="wrapper" bind:this={wrapper}>
+	<section>
+		<div class="cards">
+			{#each agendaItems as { event, date, address, price, image }}
+				<div>
```
Comment (ju5tu5) [inline src/lib/molecules/agenda-carousel.svelte:45]: why the ```<div>```'s?

File: src/lib/molecules/agenda-carousel.svelte [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+						cardDate={date}
+						cardAddress={address}
+						cardPrice={price}
+						cardImage={image}
+					/>
+				</div>
+			{/each}
+		</div>
+
+		<!-- <span class="carrousel-indicator">
```
Comment (ju5tu5) [inline src/lib/molecules/agenda-carousel.svelte:93]: same...

File: src/routes/+page.svelte [hunk 5]
```diff
@@ -1,17 +1,21 @@
...
 <Header />
 <HeroSlider />
 <Footer />
-<Button
-	buttonText="Bekijk de Bieb"
-	buttonBackground="#4E7141"
-	svgFill="#F9E9C2"
-	buttonColor="#F9E9C2"
+<AgendaCarousel
+	agendaItems={[
```
Comment (ju5tu5) [inline src/routes/+page.svelte:10]: Please put the object literal inside the ```<script>``` section and use a ```const``` name to refer to it..

**Review by ju5tu5 — CHANGES_REQUESTED** [review]
Few tweaks and it's perfect.. nice work!

File: src/lib/molecules/agenda-card.svelte [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+		<time> {cardDate} </time>
+		<address>{cardAddress}</address>
+		<span> {cardPrice} </span>
+	</div>
+	<div class="button">
```
[the remark this answers is not in the document]
Comment (IvarSchuyt) [reply src/lib/molecules/agenda-card.svelte:14]: the div is used for positioning, this way I can use 'align self' on a sveltekit component

File: src/lib/molecules/agenda-carousel.svelte (hunk 2 above)
Comment (IvarSchuyt) [reply src/lib/molecules/agenda-carousel.svelte]: removed

File: src/lib/molecules/agenda-carousel.svelte (hunk 3 above)
Comment (IvarSchuyt) [reply src/lib/molecules/agenda-carousel.svelte]: changed to ul and li

File: src/lib/molecules/agenda-carousel.svelte (hunk 4 above)
Comment (IvarSchuyt) [reply src/lib/molecules/agenda-carousel.svelte]: fixed

File: src/routes/+page.svelte (hunk 5 above)
Comment (IvarSchuyt) [reply src/routes/+page.svelte]: changed

**Pull request closed without merging**